cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -789,8 +789,8 @@ SwiftGenerator::writeOpDocSummary(IceInternal::Output& out, const OperationPtr&
 
     if (!doc.misc.empty())
     {
-        out << nl;
-        writeDocLines(out, doc.misc, false);
```

Comment:
This is the part that fixes #2088.

The `false` parameter here meant: "don't put a triple slash in front of this line`.
So this code would generate:
```
    /// - parameter current: `Ice.Current` - The Current object for the dispatch.
    @remarks This is a test!
```
With no leading triple slash, which rightfully angered the compiler.

Now, we generate:
```
    /// - parameter current: `Ice.Current` - The Current object for the dispatch.
    ///
    /// @remarks This is a test!
```